solutions/HodlCapsule.sol

```diff
@@ -2,16 +2,22 @@ pragma solidity 0.4.24;
 
 
 contract Capsule {
+
+    //declared variables: owner_, amount_ and unlocktime_
     address public owner_;
     uint256 public amount_;
     uint public unlockTime_;
 
+    // create a constructor function that makes this contract payable
+        // it should assign the variables you just declared
     constructor() public payable {
         owner_ = msg.sender;
         amount_ = msg.value;
         unlockTime_ = now + 3 minutes;
     }
 
+    // create a send function that will send the ether back to the owner once the unlocktime has been reached
+        // use the selfdestruct function to revert the contract
     function send() external {
```

Comment:
upon review I think we should rename this to avoid a collision with the low level solidity `send` method.  Perhaps `withdraw` ?